README.md

```diff
@@ -1 +1,141 @@
-# Magembe-portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Magembe-Portfolio] <a name="Magembe-Portfolio"></a>
+A simple Portfolio html website of Magembe Ssekayi
+
+**[Magembe-Portfolio]** is a simple portfolio website of Magembe Ssekkayi
+## 🛠 Built With <a name="built-with"></a>
+
+HTML,CSS
+
+### Tech Stack <a name="tech-stack"></a>
+HTML, CSS
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add Linters]**
+- **[Add some html]**
+- **[Add some background images]**
+- **[Add some styles]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
```

Comment:
- [optional] the live demo link is broken. It will be better if you remove the live demo section, since you don't yet have a deployed version of the app